src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2165,13 +2193,13 @@ internal override BaseClientSessionTransportManager CreateClientSessionTransport
 
         #endregion
 
-        #region Internal Methods
+        #region Public Methods
 
         /// <summary>
         /// StartSSHProcess.
         /// </summary>
         /// <returns></returns>
-        internal int StartSSHProcess(
+        public int StartSSHProcess(
```

Comment:
I need access in my SSHConnection example implementation, where I wrap this object to avoid duplicating existing code.

Reply:
I personally would be wary of making this public, IMO it's not really that flexible enough to be useful publicly as it relies on a lot of assumed information specific to PSRemoting. By making it public you essentially need to keep that code in place as is without changing it in the future. It might be better off to just duplicate the code for your test connection rather than making it public (alongside the other methods turned public in this class).

Reply:
The `StartSSHProcess` method is complex, requiring different build and distributions depending on target platform, and is not worth duplicating here.  Instead, I will remove the SSHConnection repo and replace it with the NamedPipeConnection repo, and make these methods internal again.

But I actually don't think making this API public is that big of a deal.